client/src/components/AppointmentCreateForm.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+
+import "react-datetime/css/react-datetime.css";
+
+import React, { useState } from "react";
+import { gql, useMutation } from '@apollo/client';
+
+import Datetime from "react-datetime";
+import { FETCH_APPOINTMENTS } from "./PickupList";
+import { useHistory } from "react-router-dom";
+
+const CREATE_APPOINTMENT = gql`
+  mutation createAppointment($startTime:Int! $endTime:Int! $address:String! $note:String! ){
+    createAppointment(startTime:$startTime, endTime:$endTime, address:$address, note:$note){
+      id,
+      location{
+        id
+        address
+        note
+      },
+      startTime,
+      endTime
+  }}
+`;
+
+const AppointmentCreateForm: () => React$Node = () => {
+
+  const history = useHistory();
+  const backToLanding = () => {
+    history.push("/");
+  };
+
+  const [newFromTime, setNewFromTime] = useState('');
+  const [newToTime, setNewToTime] = useState('');
+  const [newAddress, setNewAddress] = useState('');
+  const [newNote, setNewNote] = useState(' ');
+
+  const [ createAppointment,
+          {loading:mutationLoading, error: mutationError},
+   ] = useMutation(CREATE_APPOINTMENT, {
+    refetchQueries: [{query: FETCH_APPOINTMENTS}],
+    onCompleted: () => backToLanding()
+  });
+
+  const fromTimeInputProps = {
+    placeholder: "Click to open time picker",
+    id: "pickupFromTime",
+    required: "required"
+  };
+  const toTimeInputProps = {
+    placeholder: "Click to open time picker",
+    id: "pickupToTime",
+    required: "required"
+  };
+  
+  var startTime = new Date(newFromTime);
```

Comment:
`Date` (javascript builtin class) and moment.js are two equivalent ways of dealing with timestamp values in javascript, you need to pick one of the two and stick to it, rather than using a mix of both. [this post](https://www.freecodecamp.org/news/the-ultimate-guide-to-javascript-date-and-moment-js/) covers both and [this stackoverflow question](https://stackoverflow.com/questions/29606643/difference-between-moment-js-functions-and-javascript-date-functions) can give you some info on how to choose between them.

Reply:
I am going to use `moment.js` only. Both of them work for us so far.